### PR TITLE
Set up NODE_DEBUG and longjohn to debug network issues

### PR DESCRIPTION
**Pull request by axmb**

### CODE REVIEW

File: data-serving/data-service/Dockerfile [hunk 1]
```diff
@@ -38,6 +38,10 @@ USER node
 # Run with node env set to production.
 ENV NODE_ENV production
 
+# TODO: Remove this pending https://github.com/globaldothealth/list/issues/961.
+# Run with node debug mode set to http, net, and tls.
+ENV NODE_DEBUG http,net,tls
```
Comment (attwad) [inline data-serving/data-service/Dockerfile:43]: you can set that up in the data.yaml and curator.yaml files instead of hardcoding those in the dockerfile

**Review by attwad — APPROVED** [review]
we need an mpm package for printing stack traces </nervous laughter>

File: data-serving/data-service/Dockerfile (hunk 1 above)
Comment (axmb) [reply data-serving/data-service/Dockerfile]: Sure -- does this LGTU?

**Review by attwad — APPROVED** [review]

File: data-serving/data-service/Dockerfile (hunk 1 above)
Comment (attwad) [reply data-serving/data-service/Dockerfile]: perfect, thanks

**Pull request merged**